test/Microsoft.ML.Tests/Transformers/SelectColumnsTests.cs

```diff
@@ -57,7 +58,6 @@ void TestSelectKeep()
             Assert.True(foundColumnA);
             Assert.Equal(0, aIdx);
             Assert.False(foundColumnB);
-            Assert.Equal(0, bIdx);
```

Comment:
>Assert.Equal(0, bIdx); [](start = 12, length = 22)

why all these changes in file?

Reply:
These tests verify that, if `TryGetColumnIndex` returns `false`, it also fills column index with 0. This was never a contract (and in fact I now fill with -1). So I removed the asserts.

---
In reply to: [235494346](https://github.com/dotnet/machinelearning/pull/1680#discussion_r235494346) [](ancestors = 235494346)